File: Eren Yeager Vs Armor Titan.c
#include <stdio.h>

int main(){
    
    int a , d=0;
    scanf("%d",&a);
    
    int b[a];
    int c[a];
    
    for(int i=0; i<a; i++){
        scanf("%d",&b[i]);
    }
    
    for(int i=0; i<a; i++){
        scanf("%d",&c[i]);
    }
    
    for(int i=0; i<a; i++){
        if(b[i]<c[i])d++;
    }
    
    
    if(d>=3) printf("YES");
    else printf("NO");
    
    return 0;
}

File: Eren Yeager Vs Armor Titan.java
import java.util.Scanner;

class Main{
    
    public static void main(String args[]){
        
        int a , d = 0;
        
        Scanner sc = new Scanner(System.in);
        
        a = sc.nextInt();
        
        int b[] = new int[a];
        int c[] = new int[a];
        
        for(int i=0; i<a; i++){
            b[i] = sc.nextInt();
        }
        
        for(int i=0; i<a; i++){
            c[i] = sc.nextInt();
        }
        
        for(int i=0; i<a; i++){
            if(b[i]<c[i])d++;
        }
        
        
        if(d>=3) System.out.println("YES");
        else System.out.println("NO");
    }
}
